kotlinTask1/src/main/kotlin/Main.kt

```diff
@@ -1,5 +1,5 @@
 fun main(args: Array<String>) {
-    println("Hello World!")
+    println("More than Hello World 2!")
 
```

Comment:
You should print "EOF" too in order to show the user that program has ended.

Reply:
-What is EOF?

Reply:
EOF is stands for "End of File"